rpc/Cargo.toml

```diff
@@ -7,29 +7,38 @@ edition = "2018"
 [dependencies]
 node-rpc-runtime-api = { path = "./runtime-api" }
 
-polymesh-primitives = { path = "../primitives", default-features = false }
+pallet-compliance-manager = { path = "../pallets/compliance-manager", default-features = false }
 pallet-pips = { path = "../pallets/pips", default-features = false }
+polymesh-primitives = { path = "../primitives", default-features = false }
 
 # Substrate
 codec = { package = "parity-scale-codec", version = "1.2.0" }
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, rev = 'c771821cae2dcb5a8808a19fae8122c0b9ae8499'}
```

Comment:
Rev is wrong on several lines. It should use `tag = "v2.0.0-rc4" `

Reply:
Well, it's the least of my worries just looking at the list of merge conflicts after yesterday's merges.